common/include/common/common_utils.hpp

```diff
@@ -587,29 +586,28 @@ static inline void warmup(function_t func, args_t&&... args) {
  * @brief Times 1 event, and returns the aggregate time.
  */
 template <typename event_t>
-static inline cl_ulong time_event(event_t&);
+static inline double time_event(event_t&);
 // Declared here, defined separately in the specific utils.hpp files
 
 /**
  * @fn time_events
  * @brief Times n events, and returns the aggregate time.
  */
-template <typename event_t>
-static inline cl_ulong time_events(std::vector<event_t> es) {
-  cl_ulong total_time = 0;
+template < typename event_t>
```

Comment:
This formatting seems a little weird, has clang-format been run on this file? If not it probably should.